contracts/registries/LicenseRegistry.sol

```diff
@@ -193,14 +193,23 @@ contract LicenseRegistry is ILicenseRegistry, AccessManagedUpgradeable, UUPSUpgr
             revert Errors.LicensingModule__LicenseTermsNotFound(licenseTemplate, licenseTermsId);
         }
 
+        LicenseRegistryStorage storage $ = _getLicenseRegistryStorage();
```

Comment:
why move this line here?  shouldn't it be at 208?

Reply:
It should be by accident. moved back.